tests/integration/indicator-network-test-base.cpp

```diff
@@ -604,6 +605,29 @@ QString IndicatorNetworkTestBase::createVpnConnection(const QString& id,
     return path.path();
 }
 
+QString IndicatorNetworkTestBase::importVpnConnection()
+{
+    nmofono::connection::ActiveConnectionManager::SPtr actConnManager =
+        make_shared<nmofono::connection::ActiveConnectionManager>(
+            QDBusConnection::systemBus()
+        );
+    nmofono::vpn::VpnManager::SPtr vpnManager = make_shared<nmofono::vpn::VpnManager>(
+        actConnManager,
+        QDBusConnection::systemBus()
+    );
+    const QString& ovpn_sample("sample.ovpn");
+
+    auto reply = vpnManager->importConnection(
+        nmofono::vpn::VpnConnection::Type::openvpn,
+        ovpn_sample
+    );
```

Comment:
For this to work, I think that `network-manager-openvpn` also needs to be added to `Build-Depends` in `debian/control` otherwise the plugin will fail to load.

Reply:
adding, thanks!

Reply:
> The test failed because the plugin couldn't be loaded, and so it threw an exception.

Thanks for the hint, though I'm afraid there is something else, since after loading the plugin and running the code, the test expects to find one element that is the new connection, and another which is the config wheel to modify it. These elements doesn't turn up.